deployer/src/slice-reader.ts

```diff
@@ -76,7 +76,7 @@ export async function fetchSlice(sliceName: string): Promise<string> {
     return ''
   }
   debug('have valid download response')
-  const zip = new Zip(response[0])
+  const zip = new Zip(response)
```

Comment:
This change requires a coordinated change to `@nimbella/sdk`.   The version currently tagged as `dev` needs to be retagged as `latest`.   There is a corresponding PR in `nimbella-sdk-nodejs` covering the changes there.   Note that this code only executes inside runtime containers (to support remote build) and so merging this branch will not immediately break anything.  Once this code is in production, then runtimes should not be rebuilt and redeployed in production until the SDK version has been adjusted.